hil-test/tests/i2c.rs

```diff
@@ -44,6 +44,13 @@ mod tests {
         Context { i2c }
     }
 
+    #[test]
+    #[timeout(3)]
+    fn empty_write_returns_ack_error_for_unknown_address(mut ctx: Context) {
+        assert_eq!(ctx.i2c.write(0xab, &[]), Err(Error::AckCheckFailed));
+        assert_eq!(ctx.i2c.write(0x77, &[]), Ok(()));
```

Comment:
At first I read this thinking, shouldn't this fail too? But 0x77 is the real device connected to the HIL device. Perhaps 0x77 should go behind a const or an extra comment should go here for clarity.